src/main/java/com/sourcegraph/javagraph/GenericProject.java

```diff
@@ -15,7 +14,7 @@ public GenericProject(SourceUnit unit) {
     }
 
     @Override
```

Comment:
This class (`GenericProject`) wasn't modified much, but it could also use a short docstring at the top.